frontend/vue/components/Syllabus/SyllabusView.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <section class="syllabus-view">
+    <h1 class="syllabus-view__title">
+      {{ syllabus.name }}
+    </h1>
+    <h2 class="syllabus-view__section-title">
+      General Information
+    </h2>
+    <SyllabusGeneralInformation class="syllabus-view__general-information" :syllabus="syllabus" />
+    <div
+      v-for="(course, idx) in syllabus.courseList"
+      :key="`${idx}-${course.title}`"
+      class="syllabus-view__course"
+    >
+      <div class="syllabus-view__course__title">
+        {{ course.title }}
+      </div>
+      <p class="syllabus-view__course__description">
+        {{ course.description }}
+      </p>
+      <div class="syllabus-view__course__units-title">
```

Comment:
We shouldn't show **Chapters** if there are no units

Suggested implementation:
- Adding `v-if`
- If `course.unitList` isn't empty, then show **Chapters**